### PR TITLE
Switch to a home build

### PR DESCRIPTION
**Pull request by pvalsecc**

### CODE REVIEW

File: builder/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:17.04
+MAINTAINER Camptocamp<info@camptocamp.com>
```
Comment (sbrunner) [inline builder/Dockerfile]: Deprecated ...

File: server/.dockerignore [hunk 2]
```diff
@@ -0,0 +1 @@
+build
```
Comment (sbrunner) [inline server/.dockerignore]: Not Dockerfile?

File: server/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,43 @@
+FROM ubuntu:17.04
+
```
Comment (sbrunner) [inline server/Dockerfile]: Missing maintainer

**Comment by pvalsecc** [comment]
> How to build a specific MapServer version?

You need to create a new branch, for that. 

> Shouldn't we document that we will create a version 7.0, 7.2, ..., and mot 7.0.0, 7.0.1, ...

I could catch the label a generate a specific version, but then this specific version wouldn't be built every night for security upgrades. I prefer not.

**Pull request merged**